src/lib/UtapiClient.js

```diff
@@ -554,163 +623,94 @@ export default class UtapiClient {
     * @param {callback} callback - callback to call
     * @return {undefined}
     */
-    _pushMetricPutObject(params, timestamp, action, log, callback) {
+    _genericPushMetricPutObject(params, timestamp, action, log, callback) {
```

Comment:
You might include in the JSDoc that this also updates counter for CopyObject action.